README.md

```diff
@@ -68,3 +68,104 @@
 | tcp\_listener\_arn | The ARN of the TCP listener |
 
 <!--- END_TF_DOCS --->
+
```

Comment:
all looks good to me, except I just noticed that the readme tfdocs has been duplicated. Can you check this out?